apps/docs/src/development/index.md

```diff
@@ -10,6 +14,10 @@ This library is currently in early / active development.
 
 </div>
 
+## Nuxt
```

Comment:
If we mention Nuxt that prominently on the getting started page, should we maybe also point out that the installation guide is to be used inside a Vue project?

Reply:
What do you mean by that? :)

Reply:
nevermind, we prominently mention it's a vue lib on the landing page ^^